css/properties/-webkit-line-box-contain.json

```
@@ -5,7 +5,8 @@
         "__compat": {
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": "≤15",
```

See: https://github.com/caugner/css-properties/commit/31ff6c8ee553e428fd2d1be7b32f219ec3e6ac36

```suggestion
              "version_added": "12",
```